website/templates/include/profile/social.mako

```diff
@@ -1,86 +1,106 @@
 <script id="profileSocial" type="text/html">
 
     <div data-bind="if: mode() === 'edit'">
-
         <form role="form" data-bind="submit: submit">
-
-            <div class="form-group">
-                <label>Personal Site</label>
-                <div class="input-group">
-                <span class="input-group-addon"><i class="glyphicon glyphicon-globe"></i></span>
-                <input class="form-control" data-bind="value: personal" placeholder="http://personalsite.com"/>
+            <label>Your Websites</label>
+            <div data-bind="sortable: {
+                        data: profileWebsites,
+                        options: {
+                            handle: '.sort-handle',
+                            containment: '#containDrag'
+                        }
+                    }">
+
+                <div>
+                    <div class="sort-handle">
+                        <i title="Click to Remove" class="btn text-danger pull-right  fa fa-times fa" data-bind="click: $parent.removeWebsite"></i>
+                        <div class="input-group" >
+                            <span class="input-group-addon"><i title="Drag to Reorder"  class="glyphicon glyphicon-menu-hamburger"></i></span>
```

Comment:
Prefer fontawesome to glyphicons. This might be what you need here: https://fortawesome.github.io/Font-Awesome/icon/bars/